content/influxdb/v2.0/monitor-alert/custom-checks.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Create custom checks
+seotitle: Custom checks
+description: >
+  Create custom checks with a Flux task.
+menu:
+  influxdb_2_0:
+    parent: Monitor & alert
+weight: 201
+v2.0/tags: [alerts, checks, tasks, Flux]
+---
+
+In the UI, you can create two kinds of [checks](/influxdb/v2.0/reference/glossary/#check):
+[`Threshold check`](/influxdb/v2.0/monitor-alert/checks/create/#threshold-check) and
+[`Deadman check`](/influxdb/v2.0/monitor-alert/checks/create/#deadman-check).
+
+Using a Flux task, you can create a custom check that provides a couple advantages:
+
+- Set up custom criteria for your alert (other than `threshold` and `deadman`).
+- Share custom checks across distributed instances.
+
+## Create a task
+
+1. In the InfluxDB UI, select **Tasks** in the navigation menu on the left.
+
+    {{< nav-icon "tasks" >}}
+
+2. Click **{{< icon "plus" >}} Create Task**, and then select **New Task**.
+3. In the **Name** field, enter a descriptive name,
+   and then enter how often to run the task in the **Every** field (for example, `10m`).
+   For more detail, such as using cron syntax or including an offset, see [Task configuration options](/influxdb/v2.0/process-data/task-options/).
+4. Enter the Flux script for your custom check, including the [`monitor.check`](/influxdb/v2.0/reference/flux/stdlib/monitor/check/) function.
+
+### Example: Monitor failed tasks
+
+Receive an alert when a task fails.
```

Comment:
Technically, this script doesn't send the alert, it only "checks" for failed tasks. You would need to combine this with a notification rule in order to get something sent to you in slack for example.